conf.d/tmux.fish

```diff
@@ -5,7 +5,7 @@ set -q fish_tmux_autostarted || set -gx fish_tmux_autostarted false
 if test -e "$HOME/.tmux.conf"
     set -q fish_tmux_config || set -gx fish_tmux_config "$HOME/.tmux.conf"
 else if test -e (set -q XDG_CONFIG_HOME || echo "$HOME/.config")/tmux/tmux.conf
```

Comment:
Do you think this line should also be changed?

Reply:
Actually, yes. I think so. I'm not sure why that evaluates to true (and made some test trying to discern the reason, still no clue), but the adding the `echo $XDG_CONFIG_HOME` seems the correct. I will update the PR with the change.

![image](https://github.com/user-attachments/assets/9ec72669-719c-4b8c-aa51-dc252e36f405)

Reply:
Thanks!